packages/ckeditor5-dev-bundler-webpack/lib/utils/getwebpackes6config.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const path = require( 'path' );
+const BabiliPlugin = require( 'babili-webpack-plugin' );
+
+/**
+ * @param options
+ * @param {String} options.entryPoint An entry point which will be compiled.
+ * @param {String} options.destinationPath A path where compiled file will be saved.
+ * @param {String} options.moduleName A name of exported module after compilation.
+ * @param {String} options.cwd Current work directory. Required for searching the modules.
+ * @returns {Object}
+ */
+module.exports = function getWebpackES6Config( options ) {
+	return {
+		devtool: 'cheap-source-map',
+
+		entry: options.entryPoint,
+
+		output: {
+			path: options.destinationPath,
+			filename: 'ckeditor.es6.js',
```

Comment:
We redefined a bit the two types of builds: https://github.com/ckeditor/ckeditor5/issues/408#issuecomment-287575180